mmcv/fileio/file_client.py

```diff
@@ -36,6 +47,7 @@ def __init__(self, path_mapping=None):
         except ImportError:
             raise ImportError('Please install ceph to enable CephBackend.')
 
+        warnings.warn('CephBackend is deprecated using PetrelBackend instead')
```

Comment:
is deprecated -> will be deprecated, using xxx -> please use PetrelBackend instead